examples/banananation/src/main/java/gov/nasa/jpl/aerie/banananation/activities/BiteBananaActivity.java

```diff
@@ -18,6 +18,11 @@
  */
 @ActivityType("BiteBanana")
 public final class BiteBananaActivity {
+
+  /**
+   * The amount by which to reduce /fruit
+   * @aerie.unit number
```

Comment:
Just playing around a bit - any thoughts on whether I'm "holding it wrong", or if it's worth altering the behavior here?

```java
/**
 * The amount by which to reduce /fruit
 * @aerie.unit number
 *
 * Other information that I wouldn't expect to be captured in the unit.
 */
@Parameter
public double biteSize = 1.0;
```

```java
@Override
public Map<String, String> getParameterUnits() {
  return Map.ofEntries(
          Map.entry("biteSize", "number\n"
              + "\n"
              + "Other information, that I wouldn't expect to be captured in the unit."));
}
```

Reply:
This was just an oversight in implementation, I didn't expect that users would add text after the unit annotation but that's perfectly reasonable. I think the expected behavior here is that we'd only capture `@aerie.unit number`. Do you agree?

Reply:
I concur :+1: